src/content/en/2022/javascript.md

```diff
@@ -6,7 +6,7 @@ description: JavaScript chapter of the 2022 Web Almanac covering the usage of Ja
 authors: [malchata, ibnesayeed]
```

Comment:
@malchata can you suggest a bio we can include at the bottom of the chapter? Last year's for reference: https://almanac.httparchive.org/en/2021/javascript#authors

Reply:
This has been my usual bio:

"[Jeremy Wagner](https://jlwagner.net/) is a technical writer for Google on performance and Core Web Vitals. He has also written for A List Apart, CSS-Tricks, and Smashing Magazine. Jeremy will someday relocate to the remote wilderness where sand has not yet been taught to think. Until then, he continues to reside in Minnesota’s Twin Cities with his wife and stepdaughters, bemoaning the existence of strip malls."

Reply:
Thanks!